Gruntfile.js

```diff
@@ -43,8 +43,8 @@ module.exports = function(grunt) {
 
     _accept_runner: '_mocha',
     _accept_args: '-A -u exports --recursive -t 10000 ./test/setup_accept.js ./test/accept',
-    accept: ['mongo ./test/setup_mongo.js', '<%= _accept_runner %> <%= _accept_args %>'],
-    accept_cover: ['mongo ./test/setup_mongo.js', 'istanbul cover --dir cov-accept <%= _accept_runner %> -- <%= _accept_args %>'],
+    accept: ['node ./test/setup_mongo_node.js', '<%= _accept_runner %> <%= _accept_args %>'],
```

Comment:
I tried running this locally, and I hit hit issues if my MongoDB was too old. It seems from trying out the tags [on dockerhub](https://hub.docker.com/_/mongo/) that at least MongoDB 3.4 is required for something in the `setup_mongo_node.js` script. Do you think this is reasonable, or should it be changed to work with older versions? If it's reasonable, should we add something to the readme to mention it?

Reply:
I think I solved that. Should have been using err.code === 18 from the start, instead of checking err.codeName.